package.json

```diff
@@ -175,19 +173,6 @@
         "clean:cache:modules"
       ]
     },
-    "check": {
```

Comment:
Depending on where we're heading with wireit (e.g. if we're not removing it) we might want to keep `check` (but fine to nuke `check:ci`) for developers to be able to run `pnpm check` in localdev.

Reply:
Oh, and here are the existing instances of `pnpm run check` and `pnpm check` -- https://github.com/search?q=repo%3AFormidableLabs%2Fvictory+%28%22pnpm+run+check%22+OR+%22pnpm+check%22%29&type=code